packages/atomic/src/index.html

```diff
@@ -58,7 +58,7 @@
       </div>
       <div class="flex flex-col lg:flex-row">
         <atomic-facet-manager class="flex flex-row flex-wrap justify-center mx-2 mb-3 lg:flex-col lg:w-80">
-          <atomic-facet class="mr-2 mt-2" field="author" label="Authors"></atomic-facet>
+          <atomic-facet class="mr-2 mt-2" field="author" label="Authors" enable-facet-search></atomic-facet>
```

Comment:
If the default is `true`, then the `enable-facet-search` would no be necessary here.